api/Credential.json

```diff
@@ -139,19 +139,25 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Credential/name",
           "support": {
             "webview_android": {
-              "version_added": "51"
+              "version_added": "51",
+              "version_removed": "52",
```

Comment:
Out of curiosity, where did the 52 number come from? I couldn't figure it out from the links.

Reply:
I remember that it was a bit tricky, because removed lines don't show up in the blame view, but here's the Chromium diff that removes `name` and `iconURL` from `Credential` (and extracts it into a new super-interface `SiteBoundCredential`):
https://chromium.googlesource.com/chromium/src/+/be6cc81fc74f63c8e134525bbf15ea966c8403d5%5E%21/#F5

And I (hopefully) verified the `version_added` with this blame:
https://chromium.googlesource.com/chromium/src/+blame/3df6ff1f5b8a0732469c57e56c805aefde17ebfe/third_party/WebKit/Source/modules/credentialmanager/Credential.idl